src/codeflare_sdk/ray/cluster/cluster.py

```diff
@@ -744,6 +809,20 @@ def _create_resources(yamls, namespace: str, api_instance: client.CustomObjectsA
     )
 
 
+def _apply_resources(
```

Comment:
Can you rename this to make it clear that it is specific to ray clusters

Reply:
done